lua/entities/ent_jack_gmod_ezmachine_base.lua

```diff
@@ -147,6 +147,20 @@ if(SERVER)then
 		JMod.Hint(ply, self.ClassName)
 		return ent
 	end
+	function ENT:Initialize()
+	if(self:CustomInit())then
```

Comment:
ayooo we can't do that
you need to check if the field exists before assuming it to be a func and calling it
if self.CustomInit then

Reply:
Oh, right, I forgot that was the proper way to do it.